frontend/packages/core/src/Input/radio-group.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import {
+  FormControl as MuiFormControl,
+  FormControlLabel,
+  FormLabel as MuiFormLabel,
+  Radio,
+  RadioGroup as MuiRadioGroup,
+} from "@material-ui/core";
+import styled from "styled-components";
+
+const FormLabel = styled(MuiFormLabel)`
+  ${({ theme }) => `
+  && {
+    color: ${theme.palette.text.primary};
+    font-weight: bold;
+  }
+  `}
+`;
+
+const StyledRadioGroup = styled(MuiRadioGroup)`
+  ${({ ...props }) => `
+  display: flex;
+  width: 100%;
+  max-width: ${props["data-max-width"] || "500px"};
+  `}
+`;
+
+const FormControl = styled(MuiFormControl)`
+  ${({ theme, ...props }) => `
+  display: flex;
+  width: 100%;
+  margin: 16px 0;
+  max-width: ${props["data-max-width"] || "500px"};
+  .MuiInput-underline:after {
+    border-bottom: 2px solid ${theme.palette.accent.main};
+  }
+  `}
+`;
```

Comment:
```suggestion
const FormControl = styled(MuiFormControl)`
  ${({ theme, ...props }) => `
  display: flex;
  min-width: fit-content;
  width: ${props["data-max-width"] || "500px"};
  `}
`;
```

Reply:
A committed your suggestion BUT I left the `margin: 16px 0` part so that `RadioGroup`'s margins are similar to `Select`'s and `TextField`'s margins